### PR TITLE
DX-338 Update `as` `get` and `wait`

### PR DESCRIPTION
**Pull request by mccataldo**
Closes DX-338

### CODE REVIEW

File: content/api/commands/as.md [hunk 1]
```diff
@@ -58,14 +58,21 @@ it('disables on click', () => {
...
+// `PUT` requests on the `/users` endpoint will be stubbed with
+// the `user` fixture and be aliased as `editUser`
+cy.intercept('PUT', '/users', { fixture: 'user' }).as('editUser')
+// we'll assume submitting `form` triggers a matching request
 cy.get('form').submit()
-cy.wait('@putUser').its('url').should('contain', 'users')
+// once a response comes back from the `editUser`
+// this `wait` will resolve with the subject containing `url`
+cy.wait('@editUser').its('url').should('contain', 'users')
 ```
```
Comment (cowboy) [inline content/api/commands/as.md:72]: Would it be possible to put linebreaks before the comments on line 67 and 71? Interestingly enough, GH won't let me create a suggestion here.

File: content/api/commands/as.md [hunk 2]
```diff
@@ -58,14 +58,21 @@ it('disables on click', () => {
 
 ### Route
```
Comment (cowboy) [inline content/api/commands/as.md:59]: Should we change the header from `Route` to `Intercept`?

**Review by cowboy — CHANGES_REQUESTED** [review]

File: content/api/commands/as.md (hunk 1 above)
Comment (mccataldo) [reply content/api/commands/as.md:72]: OK I added whitespace

File: content/api/commands/as.md (hunk 2 above)
Comment (mccataldo) [reply content/api/commands/as.md:59]: It depends on their definitions which are currently unclear.

**Comment by mccataldo** [comment]
@cowboy FYI, this is still a WIP.

**Comment by mccataldo** [comment]
@cowboy We don't appear to be documenting using `get` with route or intercept aliases so I'm not sure there's a use case. Do you want to explore that as part of this issue or a separate one?

**Comment by cowboy** [comment]
> @cowboy We don't appear to be documenting using `get` with route or intercept aliases so I'm not sure there's a use case. Do you want to explore that as part of this issue or a separate one?

Are you talking about my "Should we change the header from Route to Intercept?" comment? If so, it was just a question, I'm not sure any changes are necessary!

File: content/api/commands/wait.md [hunk 3]
```diff
@@ -122,17 +123,18 @@ cy.wait('@getBooks')
 // responded with an empty array first
 cy.get('#book-results').should('be.empty')
 
-// now re-define the /books response
-cy.intercept('/books', [{ name: 'Emperor of all maladies' }])
+// now the request (aliased again as `getBooks`) will return one book
```
Comment (cowboy) [inline content/api/commands/wait.md:126]: This comment and code line are causing horizontal scrolling at large viewport sizes. Can you tweak the text and book name just a little to help them fit?

**Comment by mccataldo** [comment]
> > @cowboy We don't appear to be documenting using `get` with route or intercept aliases so I'm not sure there's a use case. Do you want to explore that as part of this issue or a separate one?
> 
> Are you talking about my "Should we change the header from Route to Intercept?" comment? If so, it was just a question, I'm not sure any changes are necessary!

I've already made the changes anyway. I'm talking about updating `cy.get` with an intercept example. We never demonstrated this with `cy.route` so I'm not sure there's a use case, but we could explore outside of this PR.

**Review by cowboy — CHANGES_REQUESTED** [review]

File: content/api/commands/wait.md (hunk 3 above)
Comment (mccataldo) [reply content/api/commands/wait.md:126]: Done

**Review by cowboy — APPROVED** [review]

**Pull request merged**